cyences_app_for_splunk/bin/cyences_email_utility.py

```diff
@@ -269,9 +269,10 @@ def filter_recipients_if_not_in_allowed_domains(self, recipients):
             return recipients
 
 
-    def send(self, to, cc=[], bcc=[], subject='Splunk Alert', html_body=''):
+    def send(self, to, cc=[], bcc=[], subject='Cyences Alert', html_body='', is_digest=False):
```

Comment:
Let's not make any change to subject under the email util file. Pass the full subject from calling file.